docs/index.md

```
@@ -34,7 +34,8 @@ See more in How Claudie works sections.
 
 In case you are not sure where to go next, you can just simply start with our [Getting Started Guide](./getting-started/get-started-using-claudie.md) or read our documentation [sitemap](./sitemap/sitemap.md).
 
-If you need help or want to chat with us, feel free to join our slack channel[<a href="https://join.slack.com/t/claudieio-workspace/shared_invite/zt-365u5692o-hwb2IEwPHDe6U~bpXkIAdg" target="_blank" rel="noopener noreferrer">
+If you need help or want to chat with us, feel free to join our slack channel[<a href="https://kubernetes.slack.com/archives/C05SW4GKPL3" target="_blank" rel="noopener noreferrer">
   <img src="slack_logo.png" alt="Alt text" style="height:24px;margin-left: 5px;">
-</a>]()
+</a>]()(Get invite [here](https://communityinviter.com/apps/kubernetes/community))
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)